api/types/plugin.go

```diff
@@ -143,6 +142,17 @@ type PluginConfigNetwork struct {
 	Type string `json:"Type"`
 }
 
+// PluginConfigRootfs plugin config rootfs
+// swagger:model PluginConfigRootfs
+type PluginConfigRootfs struct {
+
+	// diff ids
+	DiffIds []string `json:"diff_ids"`
```

Comment:
In this vein, the problem becomes obvious when we start looking that the [godoc](https://godoc.org/github.com/docker/docker/api/types#Plugin).